lib/OpenQA/WebAPI.pm

```diff
@@ -158,7 +158,6 @@ sub startup {
     $test_r->get('/dependencies_ajax')->name('test_dependencies')->to('test#dependencies');
     $test_r->get('/investigation_ajax')->name('test_investigation')->to('test#investigate');
     $test_r->get('/infopanel_ajax')->name('test_infopanel')->to('test#infopanel');
-    $test_r->get('/module_components_ajax')->name('module_components')->to('test#module_components');
```

Comment:
I suppose these are considered internal WebUI helpers, as we also discussed recently in the context of proper API docs. Technically we're breaking the API here. Just mentioning it on the off chance that somebody objects ;-)

Reply:
I don't object :)

Reply:
None of these routes are part of the actual API accessible under the `/api/v1` prefix. (Taking [Hyrum's law](https://xkcd.com/1172) *not* into account because that would not even allow fixing obvious bugs.)

Reply:
This *did* break something though. Please see https://progress.opensuse.org/issues/72292 and I would appreciate some help there.